km_api/account/tests/serializers/test_email_verification_serializer.py

```diff
@@ -34,6 +34,30 @@ def test_save_valid_key(
     assert mock_confirm.call_count == 1
 
 
+def test_validate_duplicate(
+        email_confirmation_factory,
+        email_factory,
+        user_factory):
+    """
+    If the email has already been verified, the serializer should not be
+    valid.
+    """
+    user = user_factory(password='password')
+    email_factory(email=user.email, verified=True)
+
+    email = email_factory(email=user.email, verified=False)
+    confirmation = email_confirmation_factory(email=email)
+
+    data = {
+        'key': confirmation.key,
+        'password': 'password',
+    }
+
+    serializer = serializers.EmailVerificationSerializer(data=data)
+
+    assert not serializer.is_valid()
```

Comment:
I think this looks good. If an email has already been verified, and another user tries to verify the same email, then the new 'confirmation' object should not be valid.